CHANGELOG.md

```diff
@@ -7,7 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-- n/a
+- Added 'population_sources' and 'ubi_population'
```

Comment:
This could be a little more explicit like "Added 'population_sources' list and 'ubi_population' flag as attributes to yum repository'. May also add '###Added' heading here.

Reply:
+1